run-network/creator.py

```diff
@@ -0,0 +1,473 @@
+    image: {{ image }}
+    environment:
+      - MONGODB_USER=${MONGODB_USER:-admin}
+      - MONGODB_PASS=${MONGODB_PASS:-admin}
+      - RABBITMQ_USER=${RABBITMQ_USER:-admin}
+      - RABBITMQ_PASS=${RABBITMQ_PASS:-admin}
+      - OCRD_NETWORK_SOCKETS_ROOT_DIR=${SOCKETS_DIR:-/data/sockets}
+      - OCRD_NETWORK_LOGS_ROOT_DIR=${LOGS_DIR:-/data/logs}
+    command: |
+      /bin/bash -c "echo -e \\"
+        internal_callback_url: ${INTERNAL_CALLBACK_URL}
+        use_tcp_mets: true
+        process_queue:
+          address: ocrd-rabbitmq
+          port: 5672
+          skip_deployment: true
+          credentials:
+            username: ${RABBITMQ_USER}
+            password: ${RABBITMQ_PASS}
+        database:
+          address: ocrd-mongodb
+          port: 27017
+          skip_deployment: true
+          credentials:
+            username: ${MONGODB_USER}
+            password: ${MONGODB_PASS}
+        hosts: []\\" > /data/ocrd-processing-server-config.yaml && \\
+        ocrd network processing-server -a 0.0.0.0:8000 /data/ocrd-processing-server-config.yaml"
+    user: "${USER_ID}:${GROUP_ID}"
+    volumes:
+      - "${DATA_DIR_HOST}:/data"
+      - "${RUN_NETWORK_DIR}/ocrd-all-tool.json:/build/core/src/ocrd/ocrd-all-tool.json"
+    ports:
+      - ${OCRD_PS_PORT}:8000
+"""
+
+MONGODB_TEMPLATE = """
+  ocrd-mongodb:
+    container_name: ocrd-mongodb
+    image: mongo:latest
+    environment:
+      - MONGO_INITDB_ROOT_USERNAME=${MONGODB_USER:-admin}
+      - MONGO_INITDB_ROOT_PASSWORD=${MONGODB_PASS:-admin}
+    ports:
+      - "27018:27017"
+"""
+
+RABBITMQ_TEMPLATE = """
+  ocrd-rabbitmq:
+    container_name: ocrd-rabbitmq
+    image: rabbitmq:3-management
+    environment:
+      - RABBITMQ_DEFAULT_USER=${RABBITMQ_USER:-admin}
+      - RABBITMQ_DEFAULT_PASS=${RABBITMQ_PASS:-admin}
+    ports:
+      - "5672:5672"
+      - "15672:15672"
+"""
+
+
+DELEGATOR_PROCESSOR_TEMPLATE = """#!/usr/bin/env python
+
+from ocrd.cli import cli as ocrd_cli
+import click
+
+
+PROCESSING_SERVER_PORT = 8000
+PROCESSOR_NAME = "ocrd-cis-ocropy-binarize"
+
+
+@click.command()
+@click.option("-I", "--input-file-grp")
+@click.option("-O", "--output-file-grp")
+@click.option("-m", "--mets", help="METS to process", required=True)
+@click.option('-P', '--parameter-override',
+              help="Parameter override",
+              nargs=2,
+              multiple=True,
+              callback=lambda ctx, param, kv: kv)
+def cli(mets, input_file_grp, output_file_grp, parameter_override):
+    address = f"http://localhost:{PROCESSING_SERVER_PORT}"
+    args = [
+        "network", "client", "processing", "run",
+        PROCESSOR_NAME,
+        "--address", address,
+        "-m", mets,
+        "-I", input_file_grp,
+        "-O", output_file_grp,
+        "--block",
+        "--print-state",
+    ]
+    for (key, value) in parameter_override:
+        args.append("-P")
+        args.append(key)
+        args.append(value)
+    ocrd_cli(args)
```

Comment:
I would suggest improving these templates as follows:
1. use a `format`table string, so variables can be filled in directly, without regex matching
2. delegate to the click CLI directly, so its arg parsing and help strings can be reused

```suggestion
DELEGATOR_PROCESSOR_TEMPLATE = """#!/usr/bin/env python

from ocrd_network.cli import client_cli

run_cli = client_cli.commands['processing'].commands['run']
def callback(*args, **kwargs):
    kwargs['address'] = "http://localhost:{ps_port}"
    kwargs['block'] = True
    kwargs['print_state'] = True
    return run_cli.callback({processor_name}, *args, **kwargs)

params = [param for param in run_cli.params
          if param.name not in [
                  'processor_name',
                  'address',
                  'block',
                  'print_state',
          ]]
cli = click.Command(name={processor_name},
                    callback=callback,
                    params=params)

```